source/protobuf/session.proto

```diff
@@ -39,6 +39,7 @@ message DeviceProperties {
   string model = 2;
   string vendor = 3;
   string serial_number = 4;
+  bytes product_id = 5;
```

Comment:
I had this as a `string` at first, but that caused errors about the values containing invalid UTF-8 data when serializing a protocol buffer when I ran the system tests.

The error says to "Use the `bytes` type if you intend to send raw bytes." so I switched to that, but I'm not certain this is what we actually want.